virttest/ceph.py

```diff
@@ -220,3 +221,65 @@ def create_config_file(ceph_monitor):
         with open(ceph_cfg, 'w+') as f:
             f.write('mon_host = %s' % ceph_monitor)
     return ceph_cfg
+
+
+@error_context.context_aware
+def cephfs_mount(ceph_uri, mount_point, options=None, verbose=False, session=None):
+    """
+    Mount ceph volume to mountpoint.
+
+    :param ceph_uri: str to express ceph uri, e.g 10.73.xx.xx:6789:/
+    :param mount_point: mount point, str
+    :param options: mount options, e.g -o name=admin,secret=AQCAgd5cFyMQLhAAHaz6w+WKy5LvmKjmxx
+    :param verbose: enable verbose or not
+    :param session: mount within the session if given
+    """
+    cephfs_umount(ceph_uri, mount_point, verbose, session)
+    if not os.path.exists(mount_point):
+        try:
+            os.mkdir(mount_point)
+        except OSError as dirError:
+            logging.debug("Creation of the directory:%s failed:%s", mount_point, str(dirError))
+        else:
+            logging.debug("Successfully created the directory %s", mount_point)
+    utils_disk.mount(ceph_uri, mount_point, "ceph", options,
+                     verbose, session)
+
+
+@error_context.context_aware
+def cephfs_umount(ceph_uri, mount_point, verbose=False, session=None):
+    """
+    Umount ceph volume from mountpoint.
+
+    :param ceph_uri: str to express ceph uri, e.g 10.73.xx.xx:6789:/
+    :param mount_point: mount point, str
+    :param verbose: enable verbose or not
+    :param session: mount within the session if given
+    """
+    umount_result = 1
+    if ceph_is_mounted(ceph_uri, mount_point, verbose=verbose, session=session):
+        umount_cmd = "umount %s" % mount_point
+        if session:
+            umount_result = session.cmd_status(umount_cmd, safe=True)
+        umount_result = process.system(umount_cmd, ignore_status=False, verbose=verbose)
```

Comment:
you'd better run 'fuser -k' before unmount, you can reuse utils_disk.umount.

Reply:
fuser -k is dangerous operations, In my local test, it crash desktop and cause all console terminated

Reply:
what's your full command line?

Reply:
As I mentioned previously, the terminal was killed ,and there is no output,which shows command line that is executing. Fuser -km ,which is used in utils_disk.umount , could kill all processed ,which access certain file.